src/pretix/control/templates/pretixcontrol/auth/confirmation_code.html

```diff
@@ -0,0 +1,21 @@
+{% extends "pretixcontrol/auth/base.html" %}
+{% load bootstrap3 %}
+{% load static %}
+{% load i18n %}
+{% block content %}
+	<form class="form-signin" action="" method="post">
+        <h3>{% trans "Enter confirmation code" %}</h3>
+        {% csrf_token %}
+        <p>{{ message }}</p>
+        {% bootstrap_form_errors form type='all' layout='inline' %}
+        {% bootstrap_field form.code %}
+		<div class="form-group buttons">
+            <button type="submit" class="btn btn-primary">
+                {% trans "Okay" %}
+            </button>
+            <a href="{{ cancel_url }}" class="btn btn-default">
+                {% trans "Cancel" %}
+            </a>
```

Comment:
I think every other form has "Cancel" left and "Confirm" on the right, this one is reversed